packages/editor/src/store/actions.js

```diff
@@ -720,12 +720,14 @@ export function removeEditorPanel( panelName ) {
 /**
  * Returns an action object used to open/close the inserter.
  *
- * @param {boolean|Object} value                Whether the inserter should be
- *                                              opened (true) or closed (false).
- *                                              To specify an insertion point,
- *                                              use an object.
- * @param {string}         value.rootClientId   The root client ID to insert at.
- * @param {number}         value.insertionIndex The index to insert at.
```

Comment:
We could continue to make this backwards compatible by dispatching the new action to the block editor store _if_ the `rootClientId` or the `insertionIndex` are provided as part of `value`. You could also fire a deprecated notice in the console.

Reply:
I like this idea.

Reply:
This API has changed. It's a public API. We have to ensure it's backwards compatible.

Reply:
You can still pass these things, they just don't do anything within our implementation of the block editor. I'll add a deprecation and fire that though.

Reply:
> ...they just don't do anything

That was the bit I was concerned about because if a public API used to do something it needs to continue to do that. 

Looks like you fixed it up now 👍

Reply:
We can’t deprecate a public API/feature and suggest to use a private API instead.

It would non-actionable for consumers.

Reply:
Yeh actually we don't need to deprecate it do we? @Mamaduka is right that we can't point to a private API - great spot there.